lib/assessment.rb

```diff
@@ -35,6 +38,7 @@ def every_even(numbers)
   # entries from the words enumerable capitalized.
   def transform(words)
     # Your code goes here
+    words.each {|x| x.capitalized}
```

Comment:
Yep! It could be even shorter if you used the `&` trick.
`words.map(&:capitalize)`